### ISSUE TITLE
Should the ICDO-3 codes be related to grade concepts?

### ISSUE THREAD

**Issue by kzollove** [OHDSI/OncologyWG#570]

**Comment by tfalcs** [comment]
My comment just to clarify thoughts for me: Does grade 1 mean anything by itself or is it only interpretable when tied to therapeutic area. Isn't Grade settled already in Cancer Modifier vocab? what was the logic used there?

**Comment by rtmill** [comment]
Elaborating on this a bit:

There are specific ICDO codes that have a specific grade included within the definition of the condition concept. 
e.g. [Follicular lymphoma, **grade 2** of cerebrum; 9691/3-C71.0](https://athena.ohdsi.org/search-terms/terms/36559255)

The question can be asked another way - do we want to create relationships from these condition concepts, either hierarchical or lateral, that allow us to query for specific grades for circumstances where explicit grade observations are not captured. 

If we added these relationships, they could potentially be leveraged for:
- **phenotyping** 
  - e.g.,  Grade 1 tumors: 
    - (measurement) modifier of "grade = 1", or 
    - (condition) ICDO concept with a relation to "grade=1"
- and/or also during **ETL processes**  
  - e.g. if ICDO condition has relationship to a grade concept, create a grade modifier for that condition record

**Comment by cgreich** [comment]
Yes, that problem does exist. It also exists with Genomic attributes, they are also sometimes in the ICDO. Since we cannot pre-coordinate all grades with all conditions we probably have to do the opposite: Pull out the grades. 
Thoughts?

**Comment by rtmill** [comment]
> ...Pull out the grades. Thoughts?

Can you elaborate what you mean by pull out the grades?? 

In terms of volume, and assuming every ICDO3 has an associated "ICDO Histology", we could potentially make relationships between "ICDO Histology" and grade. Alternatively, we could use the existing relationships to the aforementioned ICDO histology concepts to identify the appropriate ICDO conditions, and then create relationships to those.

A [quick ATHENA search](https://athena.ohdsi.org/search-terms/terms/36546854) shows 22 ICDO Histology concepts with explicit mention of grade in the name, and another 10 or so that are flagged because grade is mentioned in the synonyms. I'd imagine the latter to be problematic but am unsure of the source. 

Another question arises as to what processes were used to determine if an ICDO Histology is standard or remapped to SNOMED morphology as it seems inconsistent and makes this issue a little more complicated.

**Comment by cgreich** [comment]
Well, we would just destandardize and map to the equivalent grade-less ICDO and the grade itself:

[Neuroendocrine tumor, grade 2 of body of stomach](https://athena.ohdsi.org/search-terms/terms/36546854) would be mapped to [Neuroendocrine tumor, NOS, of body of stomach](https://athena.ohdsi.org/search-terms/terms/44501523) and [Grade 2 tumor](https://athena.ohdsi.org/search-terms/terms/1634752).

Not sure what SNOMED has to do with it.
